### PR TITLE
feat: sync with WCAG 2.2

### PR DESCRIPTION
**Pull request by PassionPenguin**
### Description

- sync with WCAG 2.2
- **should we remove sub-entries?** (they are just plain copied text from the spec with seemingly little explanation)
  - : oops, these entries are from wiki-age, so may not be able to find the original author to have a comment on this 😢

### CODE REVIEW

File: files/en-us/web/accessibility/understanding_wcag/index.md [hunk 1]
```diff
@@ -21,31 +21,17 @@ Each of the links below will take you to pages that further expand on these area
...
 
 ## Legal standing
 
 This guide is intended to provide practical information to help you build better, more accessible websites. However, we are not lawyers, and none of this constitutes legal advice. If you are worried about the legal implications of web accessibility, we'd recommend that you check the specific legislation governing accessibility for the web/public resources in your country or locale, and seek the advice of a qualified lawyer.
 
 [What is accessibility?](/en-US/docs/Learn/Accessibility/What_is_accessibility) and particularity the [Accessibility guidelines and the law](/en-US/docs/Learn/Accessibility/What_is_accessibility#accessibility_guidelines_and_the_law) section provide more related information.
+
+## See also
+
+- [WCAG Overview](https://www.w3.org/WAI/standards-guidelines/wcag/)
```
Comment (estelle) [inline files/en-us/web/accessibility/understanding_wcag/index.md:36]:
```suggestion
- [WCAG: Perceivable](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable)
- [WCAG: Operable](/en-US/docs/Web/Accessibility/Understanding_WCAG/Operable)
- [WCAG: Understandable](/en-US/docs/Web/Accessibility/Understanding_WCAG/Understandable)
- [WCAG: Robust](/en-US/docs/Web/Accessibility/Understanding_WCAG/Robust)
- [WCAG text labels and names](/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names)
- [WCAG keyboard accessiblity](/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard)
- [WCAG Overview](https://www.w3.org/WAI/standards-guidelines/wcag/)
```
suggest adding all the guides

File: files/en-us/web/accessibility/understanding_wcag/index.md [hunk 2]
```diff
@@ -6,13 +6,13 @@ page-type: guide
...
+The WCAG guidelines are a set of recommendations for making web content more accessible developed by W3C's {{Glossary("WAI")}}, primarily for people with disabilities — but also for all user agents, including highly limited devices, such as mobile phones.
 
 ## The four principles
 
-WCAG is broadly broken down into four principles — major things that web content **must be** to be considered accessible (see [Understanding the Four Principles of Accessibility](https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head) for the WCAG definitions).
+WCAG is broadly broken down into four principles — major things that web content **must be** to be considered accessible (see [Understanding the Four Principles of Accessibility](https://www.w3.org/WAI/WCAG22/Understanding/intro#understanding-the-four-principles-of-accessibility) for the WCAG definitions).
 
 Each of the links below will take you to pages that further expand on these areas, giving you practical advice on how to write your web content so it conforms to the success criteria outlined in each of the WCAG 2.0 and 2.1 guidelines that further sub-divides each principle.
```
Comment (estelle) [inline files/en-us/web/accessibility/understanding_wcag/index.md:17]:
```suggestion
Each of the links below will take you to pages that further expand on these areas, giving you practical advice on how to write your web content so it conforms to the success criteria outlined in the WCAG 2.X four guiding principles.
```

Comment (estelle) [reply files/en-us/web/accessibility/understanding_wcag/index.md:17]: we talk about 2.2 as the current standard, so let's make this generic.

**Review by estelle — COMMENTED** [review]
because we're removing most of the history, i recommended moving a section with just one paragraph and merging it into the intro. 

As the intro to the 4 guides listed 2.0 and 2.1, while the new edits are about 2.2, i suggested changing it to 2.x and adding the two additional guides that aren't principles... and also avoiding the term "guide" to not confuse with the "G" of WCAG.

File: files/en-us/web/accessibility/understanding_wcag/index.md [hunk 3]
```diff
@@ -6,13 +6,13 @@ page-type: guide
 
 {{AccessibilitySidebar}}
 
-This set of articles provides quick explanations to help you understand the steps that need to be taken to conform to the recommendations outlined in the W3C Web Content Accessibility Guidelines 2.0 or 2.1 (or just WCAG, for the purposes of this writing).
+This set of articles provides quick explanations to help you understand the steps that need to be taken to conform to the recommendations outlined in the {{glossary("WCAG", "Web Content Accessibility Guidelines")}} (WCAG) 2.2.
 
-The WCAG 2.0 and 2.1 provide a detailed set of guidelines for making web content more accessible to people with a wide variety of disabilities. It is comprehensive but incredibly detailed, and quite difficult to gain a rapid understanding of. For this reason, we have summarized the practical steps you need to take to satisfy the different recommendations, with further links to more details where required.
+The WCAG guidelines are a set of recommendations for making web content more accessible developed by W3C's {{Glossary("WAI")}}, primarily for people with disabilities — but also for all user agents, including highly limited devices, such as mobile phones.
```
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/en-us/web/accessibility/understanding_wcag/index.md:11]:
```suggestion
The WCAG guidelines are a set of recommendations for making web content more accessible developed by W3C's {{Glossary("WAI")}}, primarily for people with disabilities — but also for all user agents, including some highly limited devices or services, such as digital assistants.
```

also let's bring the changes to the glossary term. (the first paragraph was kinda copied from the glossary one)

File: files/en-us/web/accessibility/understanding_wcag/index.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ page-type: guide
...
```
Comment (PassionPenguin) [inline files/en-us/web/accessibility/understanding_wcag/index.md:12]:
```suggestion

WCAG 2.2 is the most recent version of the guidelines with WCAG 2.1 and 2.0 also widely used. WCAG 3.0 is the working draft.
Newer versions of WCAG are not meant to supersede previous versions, but WAI recommends using the most recent version to ensure the best possible accessibility for your website.

```

**Review by estelle — APPROVED** [review]
Thanks! 🎉

**Pull request merged**